backend/languages/languages.py

```diff
@@ -0,0 +1,7 @@
+'''
+
+LANGUAGES = json.loads(_codes)
```

Comment:
I'd add just a few language for now, like Italian and English, no need to populate all the languages of the world :)

Reply:
done! :)